src/git.js

```diff
@@ -18,7 +18,17 @@ async function getLastCommitMessage() {
   return await exec('git log -1 --pretty=%B');
 }
 
+async function getAttemptedCommitMessage() {
+  try {
+    // This file is created by Husky to hold the contents of the commit message.
+    return await exec('cat .git/COMMIT_EDITMSG');
+  } catch (err) {
+    return '';
```

Comment:
This is just silencing the error for when the above file (`.git/COMMIT_EDITMSG`) is not present. I'm 100% for a better implementation here, if there is one.